.github/workflows/dotnet.yml

```diff
@@ -0,0 +1,139 @@
+        Start-Process dotnet -ArgumentList "run -v m -c ${{ env.buildConfiguration }} --launch-profile CDR.Register.Discovery.API.Pipeline"
+        
+        cd ${{ github.workspace }}\Source\CDR.Register.SSA.API
+        Start-Process dotnet -ArgumentList "run -v m -c ${{ env.buildConfiguration }} --launch-profile CDR.Register.SSA.API.Pipeline"
+        
+        cd ${{ github.workspace }}\Source\CDR.Register.Status.API
+        Start-Process dotnet -ArgumentList "run -v m -c ${{ env.buildConfiguration }} --launch-profile CDR.Register.Status.API.Pipeline"
+        
+        cd ${{ github.workspace }}\Source\CDR.Register.IdentityServer
+        Start-Process dotnet -ArgumentList "run -v m -c ${{ env.buildConfiguration }} --launch-profile CDR.Register.IdentityServer.Pipeline"
+        
+        cd ${{ github.workspace }}
+        
+        Write-Host "Sleeping for 30 seconds to allow solutions to startup..."
+        Start-Sleep -s 30
+        
+        Write-Host "Listing contents of d:\a\mock-register\mock-register\cdr directory..."
+        cd d:\a\mock-register\mock-register\cdr
+        dir
+        
+        Write-Host "Basic connectivity testing..."
+        [System.Net.ServicePointManager]::ServerCertificateValidationCallback = { $True }
+        
+        try
+        {
+          Write-Host "Testing https://localhost:7000/idp/.well-known/openid-configuration..."
+          $apiReq = [System.Net.HttpWebRequest]::CreateHttp("https://localhost:7000/idp/.well-known/openid-configuration")
+          $apiRes = $apiReq.GetResponse()
+          Write-Host $apiRes.StatusCode
+        }
+        catch
+        {
+          Write-Host "Web request failed"
+          $log = Get-Content "d:\\a\\mock-register\\mock-register\\cdr\\Logs\\cdr-mr-tls-gateway.log" -Raw
+          Write-Host $log
+          
+          $log = Get-Content "d:\\a\\mock-register\\mock-register\\cdr\\Logs\\cdr-mr-identityserver.log" -Raw
+          Write-Host $log
+        }
+        
+        try
+        {
+          Write-Host "Testing https://localhost:7000/cdr-register/v1/banking/data-recipients..."
+          $apiReq = [System.Net.HttpWebRequest]::CreateHttp("https://localhost:7000/cdr-register/v1/banking/data-recipients")
+          $apiRes = $apiReq.GetResponse()
+          Write-Host $apiRes.StatusCode
+        }
+        catch
+        {
+          Write-Host "Web request failed"
+          $log = Get-Content "d:\\a\\mock-register\\mock-register\\cdr\\Logs\\cdr-mr-tls-gateway.log"
+          Write-Host $log
+        }
+        
+        Write-Host "Running the integration tests..."
+        cd ${{ github.workspace }}\Source\CDR.Register.IntegrationTests
+        ($env:ASPNETCORE_ENVIRONMENT="Pipeline") | dotnet test ${{ github.workspace }}\Source\CDR.Register.IntegrationTests\bin\Release\net5.0\CDR.Register.IntegrationTests.dll --settings ${{ github.workspace }}\Source\CDR.Register.IntegrationTests\integration.pipeline.runsettings
```

Comment:
We should use the ${{ env.runEnvironment }} var here instead of hard coding "Pipeline"

Reply:
That's been done.  I've removed all hardcoded references to Pipeline in the yml file.